blog-posts/scoped-registry.md

```diff
@@ -0,0 +1,122 @@
+//registry.working-wombats.org/:_auth=c2lyV29tYmF0OnRoZUN1ZGQxZXIh
+```
+
+>💡 Note the // without the HTTPS: at the beginning
+
+Alternatively, if you have an access token you can put the access token after `_authToken`:
+
+```
+@workingwombats:registry=https://registry.working-wombats.rocks/
+//registry.working-wombats.org/:_authToken=c2lyd29tYmF0OnRoZWN1ZGQxZXIh
+```
+
+Jeff tried it again et voilà it worked! He almost committed the new additions of the project-level .npmrc to git but then his old mentor "Sir Wombat" came into his mind:
+
+>💡 Remember that it is not a good practice to add credentials to version control e.g. if you have versioned your .npmrc file.
+
+and Jeff was asking himself...
+## But where should I put the credentials then? 🤷‍♂️
+
+If you have a project-level .npmrc file and have added it in your version control then you could do the following:
+
+**Use environment variables**
+
+we could export the authentication string as an environment variable:
+
+e.g. for linux `$ export NPM_TOKEN=<base64-string>`
+
+We can then add the env variable in our .npmrc file:
+
+```
+//registry.wombatcorp.org/:_auth=${NPM_TOKEN}
+```
+
+The npm CLI will replace this value with the contents of the NPM_TOKEN environment variable
+    
+**Add scoped registry to user-level instead of project-level**
+
+There are different .npmrc files as explained here ([npmrc](https://docs.npmjs.com/cli/v8/configuring-npm/npmrc)). 
+Each of these files is loaded, and config options are resolved in priority order. For example, a setting in the user config file would override the setting in the global config file. So we could put the scoped registry into the user config file instead of the project config file.
+
+## Conclusion
+
+Finally, Jeff could shut down his computer and leave work. Today he learned how to use scoped registries to access different registries for different scoped packages including how to use authentication. On his way home he quickly recapped what he has learned today:
+
+Packages in npm can be grouped with scopes. Scopes are preceded by a `@` followed by the scope name and the package name separated by `/` e.g. `@scope/package`. To set a specific registry for a scope you can use scoped registries. To set a scoped registry open up the .npmrc file and enter the following:
+
+```
+@scope:registry=https://<your registry>
+```
+
+where `@scope` is the scope name and `<your registry>` is the registry where the scoped packages are hosted. 
+
+If authentication is needed it is possible to add a line below the scoped registry with `_auth` and the basic authentication string for username and password (`username:password` ⇒ base64 encoded) or `_authToken` for an access token:
+
+```
+@scope:registry=https://<your registry>
+//<your registry>:_auth=<base64-string>
+```
+
+"Mhh", thought Jeff, in the end, this wasn't too hard. He put on his headphones and started hearing music and enjoyed the rest of the day.
```

Comment:
Attention: Rant ahead.

> Finally Jeff could shutdown his computer and leave work. [...] On his way home he quickly recapped what he has learned today:
[...]
He put on his headphones and started hearing music and enjoyed the rest of the day.

This feels weird. Is he using the headphones with his phone after he left work? Also, while this storytelling approach to blog posts might be appealing to some, it's not to me. Just my opinion. It seems unprofessional to me to bloat a technical blog with a story that does not add value to the technical information.

Sidenote: I'm pretty sure "started listening to music" would be the proper translation here. "started hearing music" implies that all of a sudden there is music playing from somewhere unknown.

Reply:
>  It seems unprofessional to me to bloat a technical blog with a story that does not add value to the technical information.

First of all: I think telling a story or not is a personal flavor of writing an article, it's not related to being professional or not. We are not talking about a user manual or technical documentation here. @Wompipomp is writing his first blog post so we should be open minded and not tear down the entire approach he chose 🙏

You could say the same not only about technology but also about anything that is transporting knowledge. I for myself like stories because it stretches the time you deal with a topic. This puts the topic in much deeper regions of my brain than just reading condensed facts. That's why I really don't like summary services like [blinkist](https://www.blinkist.com) for example.

Everyone has their own liking, I'm happy that you shared yours (and announced it as a rant to be prepared 😄). Mark could try out that writing style in his second blog post 🙌